src/deploy-cromwell-on-azure/Configuration.cs

```diff
@@ -26,6 +26,8 @@ public class Configuration
         public string StorageAccountName { get; set; }
         public string NetworkSecurityGroupName { get; set; }
         public string CosmosDbAccountName { get; set; }
+        public string MySQLServerName { get; set; }
+        public string MySQLServerPassword { get; set; }
```

Comment:
These two new configuration values are not documented in the table in `troubleshooting-guide.md`

Reply:
Should they ever be set by the user though? I figured they wouldn't so I wasn't going to tell people about them

Reply:
Then remove them from Configuration.

Reply:
Removed, tested it works.